ui/pages/desktop-pairing/index.scss

```
@@ -3,126 +3,39 @@
   flex-flow: column;
   align-items: center;
   padding: 0 30px 0;
+  max-width: 640px;
 
   &__countdown-timer {
-    background: #f2f3f4;
-    border-radius: 15.5px;
-    padding: 7px 0 7px 0;
-    margin: 0 32px 0 32px;
+    background: var(--color-background-default);
+    border-radius: 16px;
+    padding: 8px 0;
     font-size: 12px;
+    width: 240px;
   }
 
   &__countdown-timer-seconds {
     color: var(--color-primary-default);
   }
 
-  &__icon {
-    padding-top: 24px;
-  }
-
-  &__title {
-    font-style: normal;
-    font-weight: 700;
-    font-size: 24px;
-    line-height: 140.62%;
-    text-align: center;
-    color: #24292e;
-    padding-top: 24px;
-  }
-
-  &__subtitle,
-  &__description {
-    font-style: normal;
-    font-weight: 400;
-    font-size: 14px;
-    line-height: 140.62%;
-    text-align: center;
-    color: #000;
-    padding-top: 8px;
-    margin: 0 56px;
-  }
-
-  &__description {
-    margin: 18px 0;
-  }
-
   &__otp {
     font-style: normal;
     font-weight: 500;
     font-size: 48px;
+    line-height: 48px;
     letter-spacing: 10px;
-    color: #000;
-  }
-
-  &__buttons {
-    display: flex;
-    width: 70%;
-    justify-content: space-between;
-    margin: auto;
   }
 
   &__tooltip-wrapper {
     width: 100%;
   }
 
-  &__clickable {
-    width: 100%;
-
-    &:hover {
-      cursor: pointer;
-    }
-  }
-}
-
-.desktop-pairing-warning {
-  font-style: normal;
-  font-weight: 400;
-  font-size: 14px;
-
-  &__close-button {
-    z-index: 1050;
-    font-size: 24px;
-    cursor: pointer;
-
-    &__close::after {
-      content: '\00D7';
-      font-size: 24px;
-      cursor: pointer;
-      position: relative;
-      float: right;
-      margin-top: -8px;
-    }
-  }
-
-  &__title {
-    font-weight: bold;
-    font-size: 16px;
-  }
-
-  &__text {
-    font-size: 0.875rem;
-  }
-
-  &__link {
-    color: var(--color-primary-default);
-    display: block;
+  &__clickable:hover {
     cursor: pointer;
-    line-height: 100%;
-    font-size: 0.875rem;
-    padding: 0 0;
   }
 
-  &__warning-content {
-    border-left: 5px solid var(--color-warning-default);
-    border-right: 0;
-    border-bottom: 0;
-    border-top: 0;
-    margin: 4px;
-
-    .icon {
-      position: absolute;
-      left: 5px;
-      top: 10px;
+  &__icon {
+    path {
+      fill: var(--color-text-default);
```

You icon-default here(same hex value as text-default) but aligns with color token semantics
```suggestion
      fill: var(--color-icon-default);
```

---

Good to know, thanks for pointing out